features/step_definitions/create_menu_page.rb

```diff
@@ -16,6 +16,10 @@
       path = new_user_registration_path
     when 'restaurants' then
       path = new_restaurant_path
+    when 'edit' then
+      path = edit_restaurant_path(@restaurant)
+    else
+      path = 'You have not defined a path'
```

Comment:
It's not a good idea to set `path` with this text, instead `raise` an exception. By doing this it'll return that message to your `visit` method which will then try to navigate to `You have not defined a path`